ec2/spark_ec2.py

```diff
@@ -983,6 +969,12 @@ def real_main():
     (opts, action, cluster_name) = parse_args()
 
     # Input parameter validation
+    if "." in opts.spark_version:
```

Comment:
Will do. I asked about getting rid of the `replace("v", "")` so I wouldn't have to do that, but that didn't fly. :)